man/ocitools-validate.1.md

```diff
@@ -20,6 +21,9 @@ Validate an OCI bundle
 **--path="PATH"
    Path to bundle
 
+**--hooks-check**
+  Specify check hooks on host, default is false.
```

Comment:
I'd have gone with `--hook-paths` or some such, but I expect folks will get the idea regardless.  It's probably worth detailing what the hook checks are and why this is optional.  The only check that needs to be toggle-able is “Does the `path` exist (and point to an executable)?”, and there is other validation that you can do beyond that (e.g. “Are there equals signs in all the `env` values?”).